### PR TITLE
Clarifies higher level of engagement for cohort assignment

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2365](https://openedx.atlassian.net/browse/DOC-2365)

A team wanted to know why the number of active students in Insights '= the number of students assigned to cohorts. Tweaked the descriptions in both guides to clarify that differnt levels of engagement are considered for the two measures.

### Reviewers
- [x] Subject matter expert: @jaakana
- [ ] Doc team review (sanity check): @catong @srpearce @pdesjardins 

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [x] Squash commits

File: en_us/shared/course_features/cohorts/cohorts_overview.rst [hunk 1]
```diff
@@ -112,9 +112,9 @@ more likely to foster community feeling.
 
 If you use the automated assignment strategy, you create several "auto"
 (automated) cohorts. Learners are automatically and randomly assigned to one of
-the auto cohorts when they first view any course content, including the course
-**Discussion** page or content-specific discussion topics. In this way, each
-learner in your course is assigned to a cohort; no learner is unassigned.
+the auto cohorts when they first view any course content on the **Course** or
+**Discussion** page. In this way, each learner who engages with your course is
+assigned to a cohort; no learner is unassigned.
```
Comment (JAAkana) [inline en_us/shared/course_features/cohorts/cohorts_overview.rst]: I'm struggling a bit with the last "no learner is unassigned." 

Does either of these overload the paragraph?
 "Each learner is unassigned until this event", or "Learners who do not engage with the content remain unassigned"

Comment (lamagnifica) [inline en_us/shared/course_features/cohorts/cohorts_overview.rst]: @jaakana Let's strike the semicolon. How about,
In this way, each learner who engages with the course content or its discussion community is assigned to a cohort. No learner who engages in these ways remains unassigned.

**Comment by JAAkana** [comment]
Looks good! (:palm_tree:)

**Pull request merged**